### PR TITLE
Update VERSION to 1.2.2

### PR DESCRIPTION
**Pull request by pebrc**

### CODE REVIEW

File: config/eck.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+log-verbosity: 0
```
Comment (charith-elastic) [inline config/eck.yaml:1]: I think this file is ignored by `.gitignore` in other branches.

File: hack/manifest-gen/crd_patches/all-crds.yaml [hunk 2]
```diff
@@ -0,0 +1,2673 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (charith-elastic) [inline hack/manifest-gen/crd_patches/all-crds.yaml:1]: Git mishap?

File: hack/upgrade-test-harness/testdata/upcoming/install.yaml [hunk 3]
```diff
@@ -0,0 +1,3045 @@
+---
```
Comment (charith-elastic) [inline hack/upgrade-test-harness/testdata/upcoming/install.yaml:1]: Git mishap?

**Review by charith-elastic — COMMENTED** [review]
Looks like a couple of files snuck in due to outdated `.gitignore` in this branch

File: hack/upgrade-test-harness/testdata/upcoming/install.yaml (hunk 3 above)
Comment (pebrc) [reply hack/upgrade-test-harness/testdata/upcoming/install.yaml:1]: Oh yeah I did a `add -A` going to fast. Will fix.

**Review by charith-elastic — APPROVED** [review]

**Pull request merged**